classes/utils/DBI.class.php

```diff
@@ -293,7 +293,7 @@ public function __call($method, $args) {
             foreach($args[0] as $key => $value) {
                 if(is_array($value)) {
                     $values = array_values($value);
-                    for($i=0; $i<count($values); $i++)
+                    for($i=0, $iMax = count( $values ); $i < $iMax; $i++)
                         $args[0][$key.'___'.$i] = $values[$i];
```

Comment:
Since `array_values` is used, I think you may simplify this even more with:

	foreach($values as $i => $value)
		$args[0][$key.'___'.$i] = $value;